MapboxNavigation/CarPlayManager.swift

```diff
@@ -16,6 +16,8 @@ public enum CarPlayActivity: Int {
     case previewing
     /// The user is actively navigating along a route.
     case navigating
+    /// The user is panning the map.
```

Comment:
Can the user pan the map while navigating or previewing, or only while browsing?

Reply:
only while browsing

Reply:
@1ec5 I see what you mean. Updated the panning enum to be more specific description.